_episodes/02-cupy.md

```diff
@@ -236,5 +236,103 @@ The linear convolution is actually performed on the GPU, which is shown by a nic
 
 So this implies a speedup of a factor 104/0.731 = 142. Impressive.
 
+# Using Numba to execute Python code on the GPU
+
+[Numba](http://numba.pydata.org/) is a Python library that "translates Python functions to optimized machine code at runtime using the industry-standard LLVM compiler library". You might want to try it to speed up your code on a CPU. However, Numba also [compiles a subset of Python code into CUDA kernels](https://numba.pydata.org/numba-doc/latest/cuda/overview.html) which is what we will use here. So the idea is that we can do what we are used to, i.e. write Python code and still benefit from the speed that GPUs offer us.
+
+The code we want to run is straightforward. Let us compute all prime numbers between 1 and 10000 on the CPU and see if we can speed it up.
```

Comment:
Let's avoid using "straightforward". I know the code is simple, but it may still be complicated for someone with less mathematical background. Better introduce what the code does, and also explain what a prime number is.